spotlight-client/src/DataStore/TenantStore.test.ts

```diff
@@ -99,7 +99,16 @@ test("lock tenant if set from current domain", () => {
   reactImmediately(() => {
     expect(tenantStore.currentTenant?.id).toBe("US_ND");
     expect(tenantStore.currentTenantId).toBe("US_ND");
+    expect(tenantStore.locked).toBe(true);
   });
 
-  // TODO (#391): we shouldn't be able to change to another tenant either
+  // we shouldn't be able to switch to another tenant either
+  runInAction(() => {
+    tenantStore.currentTenantId = "US_PA";
+  });
+  reactImmediately(() => {
+    expect(tenantStore.currentTenant?.id).toBe("US_ND");
+    expect(tenantStore.currentTenantId).toBe("US_ND");
+    expect(tenantStore.locked).toBe(true);
```

Comment:
yeah it's a bit convoluted. We do not, as yet, actually have a `spotlight.recidiviz.org` domain configured for prod. For ND they did indeed configure their own DNS to point `dashboard.docr.nd.gov` at our prod Firebase environment, which has been fine because the whole site just defaults to ND as the only tenant. But now the expectation is that PA is going to do the same with `something.pa.gov`, and eventually if we get a few more states on the platform we are going to want `spotlight.recidiviz.org` to exist as well. 

And then the additional staging subdomains just let us replicate those different experiences in the pre-release environment. (The default staging domain being `spotlight-staging.recidiviz.org`, which is where you can access all the tenants at once)

Does that make sense?